src/components/tooltip/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC, ReactNode, useState, useRef } from "react"
+
+import PopUp from "./popUp"
+
+export enum TooltipPosition {
```

Comment:
To me, it's very new to have a tooltip that defines more that 4 positions, TRBL...

Reply:
Fair, it's about alignment to the parent element. Given the confusion, it might be a good idea to make it a separate prop, WDYT?

I need the variation since there's not enough space on mobile to show the tooltip centred above the parent.